src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Truck started its work");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Truck stopped its work");
+
+    }
+
+}
```

Comment:
unnecessary empty lines
```suggestion
    @Override
    public void stopWork() {
        System.out.println("Truck stopped its work");
    }
}
```